.gitmodules

```diff
@@ -854,6 +854,9 @@
 [submodule "vendor/grammars/language-webassembly"]
 	path = vendor/grammars/language-webassembly
 	url = https://github.com/Alhadis/language-webassembly
+[submodule "vendor/grammars/language-ring"]
+	path = vendor/grammars/language-ring
+	url = https://github.com/MahmoudFayed/atom-language-ring
```

Comment:
@pchaigno Thanks, I have already updated the reference commit in this pull request.